### ISSUE TITLE
[Cevelop] Can't update packages

### ISSUE THREAD

**Issue by Farigh** [Cevelop/Issues#123]
### Expected Behavior

If the Help -> Check for Update option is clicked, the update should be performed without any error.

### Actual Behavior

The update can't be performed due to multiple repositories configured : 
![image](https://user-images.githubusercontent.com/11090416/79770788-29b9d400-832e-11ea-950f-0c55bbead790.png)

![image](https://user-images.githubusercontent.com/11090416/79770853-3d653a80-832e-11ea-8ee9-c376a2e467cb.png)

There are some old versions configured (and enabled), and the last 2 contains errors in the address : 
![image](https://user-images.githubusercontent.com/11090416/79771055-8cab6b00-832e-11ea-8a55-c1b650c9505e.png)

### Cevelop Version, Operating System and Compiler

Cevelop 1.14.1 Release – 17 Feb 2020

On Windows, using a custom GCC Cross-compiler toolchain

**Comment by tcorbat** [comment]
Yes, that is an issue in the current release. We'll fix that. Until then, you can edit the update sites manually and remove the superfluous `https://` as a workaround.
